test/testkube/helm-testkube-values.yaml

```diff
@@ -889,7 +895,7 @@ testkube-api:
   # -- Slack config for the events, tests, testsuites, testworkflows and channels
   slackConfig:
   # -- default executors as base64-encoded string
-  executors: ""
+  executors: "W3sibmFtZSI6ImluaXQtZXhlY3V0b3IiLCJleGVjdXRvciI6eyJleGVjdXRvclR5cGUiOiJpbml0IiwiaW1hZ2UiOiJtY3IubWljcm9zb2Z0LmNvbS9henVyZW1vbml0b3IvY29udGFpbmVyaW5zaWdodHMvY2lkZXY6dGVzdGt1YmUtaW5pdC1leGVjdXRvciIsInR5cGVzIjpbXX19LHsibmFtZSI6Imdpbmtnby1leGVjdXRvciIsImV4ZWN1dG9yIjp7ImV4ZWN1dG9yVHlwZSI6ImpvYiIsImltYWdlIjoibWNyLm1pY3Jvc29mdC5jb20vYXp1cmVtb25pdG9yL2NvbnRhaW5lcmluc2lnaHRzL2NpZGV2OnRlc3RrdWJlLWdpbmtnby1leGVjdXRvciIsImNvbW1hbmQiOlsiZ2lua2dvIl0sImFyZ3MiOlsiLXIiLCItcCIsIi0tcmFuZG9taXplLWFsbCIsIi0tcmFuZG9taXplLXN1aXRlcyIsIi0ta2VlcC1nb2luZyIsIi0tdHJhY2UiLCItLWp1bml0LXJlcG9ydCIsIjxyZXBvcnRGaWxlPiIsIjxlbnZWYXJzPiIsIjxydW5QYXRoPiJdLCJ0eXBlcyI6WyJnaW5rZ28vdGVzdCJdLCJjb250ZW50VHlwZXMiOlsic3RyaW5nIiwiZmlsZS11cmkiLCJnaXQtZmlsZSIsImdpdC1kaXIiLCJnaXQiXSwiZmVhdHVyZXMiOlsiYXJ0aWZhY3RzIiwianVuaXQtcmVwb3J0Il0sIm1ldGEiOnsiaWNvblVSSSI6ImdpbmtnbyIsImRvY3NVUkkiOiJodHRwczovL2t1YmVzaG9wLmdpdGh1Yi5pby90ZXN0a3ViZS90ZXN0LXR5cGVzL2V4ZWN1dG9yLWdpbmtnbyJ9fX1d"
```

Comment:
do we need to update this in future? if yes, then we should cover in the README with the instructions to update.